src/nvim/indent_c.h

```diff
@@ -3,6 +3,12 @@
 
 #include "nvim/vim.h"
 
+// Find result cache for cpp_baseclass
+typedef struct {
+    int found;
+    lpos_T lpos;
+} cpp_baseclass_cache_T;
```

Comment:
Other modules don't need this, so it should go directly in `indent_c.c` as in the original patch.